hummingbird/ml/_topology.py

```diff
@@ -40,20 +45,40 @@
     DataFrame = None
 
 
+def _jit_model(torch_model, trace_input, device, extra_config):
+    """
+    Function used to convert an input pytorch model into torchscript.
+    """
+    if device != "cpu":
+        trace_input.to(device)
+    return torch.jit.trace(torch_model, trace_input).eval()
+
+
 def _get_trace_input_from_test_input(input, batch_size):
     """
     Utility function used to properly put the inputs into a format understandable by torch.
+    If a not None batch_size is passed, the function generates a tracing input of batch_size.
+    If input size % batch_size is not 0, a tracing input of the remainder is also generated.
     """
+    remainder = None
     if type(input) is tuple:
         if batch_size is not None:
             trace_input = tuple([torch.from_numpy(i)[0:batch_size, :] for i in input])
+            if len(input) > 0 and input[0].shape[0] % batch_size != 0:
+                remainder_size = input[0].shape[0] % batch_size
```

Comment:
codecov says we have no tests for line 68-69. These seem like potential problem lines, add a test?

Reply:
Alright this was not supposed to happen. I have a small bug in the tests. Fixing it.